sprite.h

```diff
@@ -62,6 +67,15 @@ class Impostor : public Sprite {
 public:
 	Impostor(const PaletteData *palette, const Point &home);
 
+	virtual void update(Context &ctx) override;
+
 protected:
 	static Bitmaps::Definition &random_bitmap();
 };
+
+class Trail : public Sprite {
```

Comment:
The name of this class is a little misleading, as it doesn't represent the whole trail, just a single sprite in the trail. Maybe it should be called `TrailSprite` or something.

Reply:
It's been renamed.